src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String description;
```

Comment:
title and description can be empty?
Film title: ""
Film description: ""
does it make any sense?